src/main/twirl/views/renderers/textAreaField.scala.html

```diff
@@ -25,11 +24,11 @@
 
 <div class='form-group @errorClass'>
     @errors.map { e =>
-        <span class="error-message">@e.err</span>
+        <span class="error-message" role="alert">@e.err</span>
     }
 
     <label for="@field.name">@field.label</label>
-    <textarea id="@field.name" class="form-control" title="@field.name" name="@field.name">@answers.get(field.name)</textarea>
+    <textarea id="@field.name" class="form-control" title="@field.name" name="@field.name" aria-label="@questions.get(field.name).map(_.text) @field.label" aria-invalid="@invalidTrueFalse">@answers.get(field.name)</textarea>
```

Comment:
I am not sure this title elements add much.  e.g. "topicsAndSpeaker" This would be better as "Topics and Speakers" (note the pural!)

Also - the label is empty. (at least on the topics and speakers page)  There should be something helpful in there.